torch/_dynamo/codegen.py

```diff
@@ -95,7 +95,7 @@ def __call__(self, value, allow_cache=True):
             value,
             (
                 TensorVariable,
-                DynamicShapeVariable,
+                SymbolicNumericalVariable,
```

Comment:
Can we use the same name as fx.experimental.symbolic_shapes uses, namely SymNode? So SymNodeVariable